src/modules/0.0.5/aws_ec2/index.ts

```diff
@@ -41,11 +44,12 @@ export const AwsEc2Module: Module = new Module({
           const client = await ctx.getAwsClient() as AWS;
           for (const instance of es) {
             if (instance.ami) {
-              const instanceId = await client.newInstance(
-                instance.name,
+              const instanceId = await client.newInstanceV2(
                 instance.instanceType,
                 instance.ami,
                 instance.securityGroups.map(sg => sg.groupId).filter(id => !!id) as string[],
+                instance.keyPairName,
```

Comment:
this wasn't passed in before which makes me think this wasn't working when creating it via IaSQL?